src/Microsoft.AspNet.Session/SessionMiddleware.cs

```diff
@@ -15,6 +15,9 @@
 
 namespace Microsoft.AspNet.Session
 {
+    /// <summary>
+    /// Enables the session state for the application.
+    /// </summary>
```

Comment:
I'm not particularly worried about having doc comments for the actual middleware types - they're almost never used directly. I think if we are going to put doc comments on these they are better off being more comprehensive than merely stating things that are likely already obvious.